cmd/fleetctl/apple_mdm.go

```diff
@@ -44,124 +52,137 @@ func appleMDMCommand() *cli.Command {
 	}
 }
 
-func appleMDMSetupCommand() *cli.Command {
+func generateCommand() *cli.Command {
 	return &cli.Command{
-		Name:  "setup",
-		Usage: "Setup commands for Apple MDM",
+		Name: "generate",
+		Flags: []cli.Flag{
+			configFlag(),
+			contextFlag(),
+			debugFlag(),
+		},
 		Subcommands: []*cli.Command{
-			appleMDMSetupSCEPCommand(),
-			appleMDMSetupAPNSCommand(),
-			appleMDMSetupDEPCommand(),
+			generateMDMAppleCommand(),
 		},
 	}
 }
 
-func appleMDMSetupSCEPCommand() *cli.Command {
-	// TODO(lucas): Define workflow when SCEP CA certificate expires.
-	var (
-		validityYears      int
-		cn                 string
-		organization       string
-		organizationalUnit string
-		country            string
-	)
+func generateMDMAppleCommand() *cli.Command {
 	return &cli.Command{
-		Name:  "scep",
-		Usage: "Create SCEP certificate authority",
+		Name:  "mdm-apple",
+		Usage: "Generates certificate signing request (CSR) and key for Apple Push Notification Service (APNs) and certificate and key for Simple Certificate Enrollment Protocol (SCEP) to turn on MDM features.",
 		Flags: []cli.Flag{
-			&cli.IntFlag{
-				Name:        "validity-years",
-				Usage:       "Validity of the SCEP CA certificate in years",
-				Required:    true,
-				Destination: &validityYears,
+			&cli.StringFlag{
+				Name:     "email",
+				Usage:    "The email address to send the signed APNS csr to.",
+				Required: true,
 			},
 			&cli.StringFlag{
-				Name:        "cn",
-				Usage:       "Common name to set in the SCEP CA certificate",
-				Required:    true,
-				Destination: &cn,
+				Name:     "org",
+				Usage:    "The organization requesting the signed APNS csr.",
+				Required: true,
 			},
 			&cli.StringFlag{
-				Name:        "organization",
-				Usage:       "Organization to set in the SCEP CA certificate",
-				Required:    true,
-				Destination: &organization,
+				Name:     "apns-key",
```

Comment:
this comment applies to `apns-key` and also `scep-cert` and `scep-key` below: per the spec, it seems like these fields are not required. Is there any reason for them to be required?

Reply:
I misunderstood how required flags work, and ended up discovering a bug. Made a fix to `app.Run(os.Args)` to handle the missing required flags error. Also removed the required flag on ones that have a default value.